src/test/java/de/tum/in/www1/artemis/exercise/modelingexercise/ModelingExerciseIntegrationTest.java

```diff
@@ -123,14 +124,7 @@ void testGetModelingExercise_asTA() throws Exception {
         exerciseUtilService.addChannelToExercise(classExercise);
 
         ModelingExercise receivedModelingExercise = request.get("/api/modeling-exercises/" + classExercise.getId(), HttpStatus.OK, ModelingExercise.class);
-        gradingCriteria = exerciseUtilService.addGradingInstructionsToExercise(receivedModelingExercise);
-        assertThat(receivedModelingExercise.getGradingCriteria().get(0).getTitle()).isNull();
-        assertThat(receivedModelingExercise.getGradingCriteria().get(1).getTitle()).isEqualTo("test title");
-
-        assertThat(gradingCriteria.get(0).getStructuredGradingInstructions()).hasSize(1);
-        assertThat(gradingCriteria.get(1).getStructuredGradingInstructions()).hasSize(3);
-        assertThat(gradingCriteria.get(0).getStructuredGradingInstructions().get(0).getInstructionDescription())
-                .isEqualTo("created first instruction with empty criteria for testing");
```

Comment:
Why did you remove these assertions?

Reply:
Those only test that our utility (`exerciseUtilService.addGradingInstructionsToExercise`) can create some grading criteria. There is no more actual interaction with the production code.

The fact that the utility produces criteria and instructions is extensively tested by using the results in other tests.